src/test/regress/sql/pg12.sql

```diff
@@ -1,405 +0,0 @@
-SHOW server_version \gset
-SELECT substring(:'server_version', '\d+')::int > 11 AS server_version_above_eleven
```

Comment:
I don't think we are supposed to remove this file? These tests seems to be new features introduced in PG 12, which should be still valid?


Removing `server_version` check makes sense

Reply:
You are right, it was a little bit of a careless decision to remove this file 😆